yoti-sdk-api/src/main/java/com/yoti/api/client/identity/NotificationRepresentation.java

```diff
@@ -0,0 +1,109 @@
+package com.yoti.api.client.identity;
+
+import static com.yoti.api.client.spi.remote.util.Validation.notNullOrEmpty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class NotificationRepresentation {
+
+    @JsonProperty(Property.URL)
+    private final String url;
+
+    @JsonProperty(Property.METHOD)
+    private final String method;
+
+    @JsonProperty(Property.VERIFY_TLS)
+    private final boolean verifyTls;
+
+    @JsonProperty(Property.HEADERS)
+    private final Map<String, String> headers;
+
+    NotificationRepresentation(String url, String method, boolean verifyTls, Map<String, String> headers) {
+        notNullOrEmpty(url, Property.URL);
+
+        this.url = url;
+        this.method = method;
+        this.verifyTls = verifyTls;
+        this.headers = headers;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public String getMethod() {
+        return method;
+    }
+
+    public boolean isVerifyTls() {
+        return verifyTls;
+    }
+
+    public Map<String, String> getHeaders() {
+        return headers;
+    }
```

Comment:
Does this needs getters? Isn't this class used to serialisation only?

Same question applies to other representations. Representations which the client needs to _produce_ but not _consume_  should not have getters.

Representations that the client only _consumes_ should have their builders private (so only jackson can use them).

Reply:
none of these representations had getters when I pushed. I had a ton of failures from PMD so I had to choose if suppress the error or add the getters. Having public getters seems the approach used everywhere even if I do not see the integrator creating an obj and use any of the getters before passing the obj as arg to the client.

Reply:
ok